gpii/node_modules/flowManager/src/CloudBasedFlowManager.js

```diff
@@ -94,16 +91,11 @@ fluid.defaults("gpii.flowManager.cloudBased.oauth2", {
     gradeNames: ["fluid.component"],
     components: {
         oauth2DataStore: {
-        // TODO: We will not have implemented a signup UI and persistence for a while. For current demonstrations, we will need
-        // to seed the implementation here with material similar to that which appears at
-        // gpii-oauth2-datastore/src/DataStoreWithSampleData.js in order to prepopulate the system with a usable collection of
-        // OAuth2 client services and users.
-        // TODO: The above comment needs to be updated when the configuration for using dbDataStore is settled.
             type: "gpii.oauth2.dbDataStore"
         },
         authServer: {
         // note that this subcomponent directly attaches express routes to our underlying express app via its direct API -
-        // e.g. at the paths /login, /authorize as well as mounting infusion at /infusion and the entire privacy UI at /privacy
+        // e.g. at the paths /access_token as well as mounting infusion at /infusion
```

Comment:
I don't think that `authServer` is serving up infusion at `/infusion` any longer.